### PR TITLE
Fix SyntaxError on Chrome < 80

### LINKED ISSUE #6: mute-broken v1.4.1 chrome 75.0
i cant mute any tabs anymore after 1.4.1 update

I do not know how to use the developer tools to check extensions
(upd: debugging background.html and devtools say 1 error "Uncaught SyntaxError: Unexpected token . debug.js:52" )

extension 1.4.1
chrome 75.0

### PR DESCRIPTION
**Pull request by markspolakovs**
Should fix #6

### CODE REVIEW

File: debug.js [hunk 1]
```diff
@@ -49,8 +49,7 @@ export function debug(...args) {
 }
 
 chrome.storage.onChanged.addListener((changes, areaName) => {
-  let val = changes["debug-mode"]?.newValue
-  if (val !== undefined) {
-    debugMode = val
+  if ("debug-mode" in changes) {
+   debugMode = changes["debug-mode"].newValue
```
Comment (keysmashes) [inline debug.js]: two spaces for indent pls :^)

File: debug.js [hunk 2]
```diff
@@ -49,8 +49,7 @@ export function debug(...args) {
 }
 
 chrome.storage.onChanged.addListener((changes, areaName) => {
-  let val = changes["debug-mode"]?.newValue
-  if (val !== undefined) {
-    debugMode = val
+  if ("debug-mode" in changes) {
```
Comment (keysmashes) [inline debug.js:52]: see #1 – should this be `changes.hasOwnProperty("debug-mode")`?

**Review by keysmashes — CHANGES_REQUESTED** [review]

File: debug.js (hunk 2 above)
Comment (markspolakovs) [reply debug.js:52]: AIUI it only matters if there are properties inherited through the prototype chain, which in this case I doubt there are (and even if they are, we'd want to pick up on it - no idea how Chrome implements this internally)

Comment (markspolakovs) [reply debug.js:52]: Chrome's docs use `for in` without `hasOwnProperty`, which leads me to think this _should_ be fine...? https://developer.chrome.com/extensions/storage#examples

Happy to change if you feel particularly strongly though

Comment (keysmashes) [reply debug.js:52]: nah, is fine

**Review by keysmashes — APPROVED** [review]

**Pull request merged**